scss/foundation-styles/type.scss

```diff
@@ -42,6 +43,10 @@ h4,
 .h4-heading {
   @include heading("Zilla Slab", 400, 0, $black, $white);
   @include scaleText(22px, 32px, 24px, 36px);
+
+  @media (min-width: $bp-md) {
```

Comment:
Same. Please separate overrides from `/foundation-styles`. Ideally we want to keep those files untouched so they can represent the copy we "borrowed" from Foundation repo.

Also to set line-height for type bases, let's utilize the mixin `scaleText`.

i.e., 
```
...
@include scaleText(22px, 32px, 24px, 28.8px);
...
```